src/components/patch-details/patch-details.tsx

```diff
@@ -5,30 +5,29 @@ import Cookies from "js-cookie";
 
 import { locale } from "../../services";
 
-import { Asset } from "@mtfh/common/lib/api/asset/v1";
 import { Patch, usePatchOrArea } from "@mtfh/common/lib/api/patch/v1";
 import {
   Button,
   Heading,
+  Spinner,
   SummaryList,
   SummaryListItem,
-  Text,
 } from "@mtfh/common/lib/components";
 
-const PatchTable = ({ patches }: { patches: Patch[] }) => {
-  const assetPatch = patches.find((patch) => patch.patchType === "patch");
-  const parentId = assetPatch?.parentId || "";
-  const parentAreaReq = usePatchOrArea(parentId);
-  const parentArea = parentAreaReq.data;
-  if (!assetPatch) return <h1>Error: Not found</h1>;
+const PatchDetailsTable = ({ assetPatch }: { assetPatch: Patch }) => {
```

Comment:
Split this primarily to ensure that the `usePatchOrArea` is only called with valid patch IDs - the Table is logically linked to the PatchDetails